### ISSUE TITLE
Database seed is incomplete

### ISSUE THREAD

**Issue by m-abrar** [SajeebChakraborty/Restaurant_Ecommerce_System_Laravel#2]
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'catagory' in 'where clause' (SQL: select * from `products` where `catagory` = regular)

**Comment by marslan633** [comment]
I'm facing the same issue. Can you please let me know, did you solve this issue?

**Comment by SajeebChakraborty** [comment]
attach this to solve this problem

**Comment by Jo766hi** [comment]
attach what?

**Comment by SajeebChakraborty** [comment]
using the sql file instead of the migration file of the laravel project.sql
file attached here....

**Comment by bapcai02** [comment]
Dear @SajeebChakraborty 

can you dump all the data out, or at least create seeders for each table

Thank you

**Comment by SajeebChakraborty** [comment]
All issue solved. Please check my repository again. Find out my sql file (named-rms2.sql) and attach it with your database. Then enjoy with my project.

**Comment by sahilchotaliya** [comment]
show 500 server error please solve this error

**Comment by Shanchitha16** [comment]
same

> SQLSTATE[42S22]: Column not found: 1054 Unknown column 'catagory' in 'where clause' (SQL: select * from `products` where `catagory` = regular)

same issue

**Comment by PicsooEU** [comment]
Hi,
same error here : 
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'catagory' in 'where clause' (SQL: select * from `products` where `catagory` = regular)

pls, dump and post a full sql file with structure and data.
this sould be a great project, thanks for sharing.
Domi.

**Comment by SajeebChakraborty** [comment]
Check rms2.sql file in root directory. It is my database...

**Comment by PicsooEU** [comment]
Yes found after !! great project, thanks for sharing - sincerly.
